operations/customers.md

```diff
@@ -24,22 +42,41 @@ Returns all customers from the specified interval according to the time filter \
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `TimeFilter` | string [Customer time filter](customers.md#customer-time-filter) | required | Time filter of the interval. |
-| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
-| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+| `CustomerIds` | array of string | optional | Identifiers of [Customer](customers.md#customer)s to use as a filter. |
+| `Emails` | array of string | optional | Emails of the [Customer](customers.md#customer)s to use as a filter. |
+| `FirstNames` | array of string | optional | First names of the [Customer](customers.md#customer)s to use as a filter. |
+| `LastNames` | array of string | optional | Last names of the [Customer](customers.md#customer)s to use as a filter. |
+| `TimeFilter` | string [Customer time filter](customers.md#customer-time-filter) | optional | Time filter of the interval. |
+| `StartUtc` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | optional | End of the interval in UTC timezone in ISO 8601 format. |
+| `Extent` | [Customer extent](customers.md#customer-extent) | optional | Extent of data to be returned. In case no extent is provided all data will be returned. |
```

Comment:
Incorrect. All consolidations we do is to avoid useless loading. If the extent is not provided we will not return everything. Now it returns, but it is only because of the compatibility (breaking change.)